packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/WorkflowSelectButton.spec.js

```diff
@@ -0,0 +1,16 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import WorkflowSelectButton from './WorkflowSelectButton'
+
+let wrapper
+
+describe('Component > WorkflowSelectButton', function () {
+  before(function () {
+    wrapper = shallow(<WorkflowSelectButton />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+})
```

Comment:
There should be tests for the URL that is constructed for the link.

Reply:
Sure. I've changed the implementation to use `withRouter` instead of `useRouter`, as I was unable to mock the router context in Enzyme, either by importing in the provider from Next or providing a context object in the wrapper options ¯\_(ツ)_/¯

Reply:
I think the latest version is supposed to support the context API. Did you try this method? https://airbnb.io/enzyme/docs/api/ShallowWrapper/getWrappingComponent.html

Reply:
I did, still came out as null :(